### ISSUE TITLE
suggest migrating this to the GACS organizational repo

### ISSUE THREAD

**Issue by semantic-nomad** [agrisemantics/gacs-qip#1]
I *think* simply changing ownership will do this rather seamlessly

**Comment by tombaker** [comment]
Do you mean the organization _account_?  The purpose of the `gacs-qip`
_repo_ is as a backend for the [GACS guidelines](
https://agrisemantics.github.io/gacs-qip/).

**Comment by semantic-nomad** [comment]
yes, sorry.  I meant move this repo to the GACS Organizational account.

However, i would split the content.  Move the website related stuff (mostly .md files) to the agrisemantics.org/website repo and put the actual qip content in a repo with the same name under the GASC org.

Does that make sense?

**Comment by tombaker** [comment]
I think it makes sense to archive the QIP project materials on a (private) archival repo somewhere, along with notes and presentations related to the old GACS WG.

Then there could be a dedicated repo just for the guidelines:
```
- 2017-11-27  11:06        833  README.md
- 2017-11-27  11:03        459  deploy.sh
- 2019-08-06  11:22        529  mkdocs.yml
d 2019-08-06  11:20        416  docs
```

If you did that, you could take the opportunity change the name of the repo to, say, `gacs-guidelines`.
